packages/lambdas/src/ihe-gateway-v2-inbound-document-query.ts

```diff
@@ -0,0 +1,53 @@
+import { ALBEvent } from "aws-lambda";
+import { InboundDocumentQueryReq, InboundDocumentQueryResp } from "@metriport/ihe-gateway-sdk";
+import * as Sentry from "@sentry/serverless";
+import { getSecretValue } from "@metriport/core/external/aws/secret-manager";
+import { processInboundDq } from "@metriport/core/external/carequality/dq/process-inbound-dq";
+import { processInboundDqRequest } from "@metriport/core/external/carequality/ihe-gateway-v2/inbound/xca/process/dq-request";
+import { createInboundDqResponse } from "@metriport/core/external/carequality/ihe-gateway-v2/inbound/xca/create/dq-response";
+import { analyticsAsync, EventTypes } from "@metriport/core/external/analytics/posthog";
+import { getEnvVarOrFail, getEnvVar } from "@metriport/core/util/env-var";
+
+const apiUrl = getEnvVarOrFail("API_URL");
+const region = getEnvVarOrFail("AWS_REGION");
+const engineeringCxId = getEnvVar("ENGINEERING_CX_ID");
+const postHogSecretName = getEnvVar("POST_HOG_API_KEY_SECRET");
+
+export const handler = Sentry.AWSLambda.wrapHandler(async (event: ALBEvent) => {
```

Comment:
I suggest we create lambdas from now on with the regular, non-Sentry handler, and always add a global try/catch. The reasoning is that our own global try/catch can send more context info to Sentry than the default one.

See `sqs-to-converter.ts` for reference.

Although, maybe we shouldn't send these to Sentry at all - inbound is our of our control and could kill our quotas. Could have a CW alarm based on the ratio of lambda invocation errors.

Reply:
Odd to have the event type as `ALBEvent` - I'd think this type would be used for requests sent from the ALB to the lambda.

Reply:
Decided to not send to sentry at all. Returning 500s for now. 

ALBEvent is the type for for ALB connecting to Lambda, which is what we have

Reply:
I thought the inbound lambda was being called straight from the API GW, no?

Reply:
Sounds good on the metrics. 
Your totally right on the AlbEvent thing. I confused myself here. I will find the proper type and replace it